src/wallet/wallet.cpp

```diff
@@ -1976,26 +1975,24 @@ bool CWallet::SetCryptedHDSeed(const uint256& seedFp, const std::vector<unsigned
         return true;
     }
 
-    /* TODO: Uncomment during PR for #3388
     {
         LOCK(cs_wallet);
         if (pwalletdbEncryption)
             return pwalletdbEncryption->WriteCryptedHDSeed(seedFp, vchCryptedSecret);
         else
             return CWalletDB(strWalletFile).WriteCryptedHDSeed(seedFp, vchCryptedSecret);
     }
-    */
+    
     return false;
 }
 
 void CWallet::SetHDChain(const CHDChain& chain, bool memonly)
 {
     LOCK(cs_wallet);
-    /* TODO: Uncomment during PR for #3388
```

Comment:
Sorry, I should have been more explicit in my comment. Please use `git revert b7f9a7ae02ffcd4bce3df9e7c9e231b0d64ed0d4` to undo this change, so that all concurrent changes are also reverted.